demo/index.html

```diff
@@ -68,6 +68,27 @@ <h1>auro-radio</h1>
         </template>
       </demo-snippet>
 
+      <p>Group w/slots</p>
+      <demo-snippet>
+        <template>
+            <auro-radio-group required>
+              <span slot="legend">Will you pick a plan?</span>
+              <auro-radio id="radio10" name="radioDemo10" value="yes">
+                <b>Lounge $300 USD</b><br>
```

Comment:
The break here isn't necessary (I tested the rendering with it removed) for the content to appear below the label. Including it lead me to initially think it was required to have the content slot not appear inline with the label.

```suggestion
                <b>Lounge $300 USD</b>
```